### PR TITLE
Changes for webinarium/symfony-lazysec 2.3+

### PR DESCRIPTION
**Pull request by webinarium**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by webinarium** [comment]
Hi guys, is the flex dead? Or is there anything else the PR should be conform with? Thanks

File: webinarium/symfony-lazysec/2.3/config/packages/lazysec.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+services:
+
+    # See https://github.com/webinarium/symfony-lazysec/wiki/User-checker
+    LazySec\Checker\GenericUserChecker:
+        autowire: true
+        public: false
```
Comment (Nyholm) [inline webinarium/symfony-lazysec/2.3/config/packages/lazysec.yaml:6]: This is redundant. It is private by default.

**Review by Nyholm — COMMENTED** [review]
Thank you for your PR. 

I've not been around the recipes for a while. Not that "Flex" and "recipes" are not the same thing. Just as a "chef" and "recipes" are not the same thing. 

Do you mind updating the PR so I can merge it?

**Comment by webinarium** [comment]
> Do you mind updating the PR so I can merge it?

Thank you for the feedback. I've updated the PR.

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**